### PR TITLE
API Assignment- 3

### PR DESCRIPTION
**Pull request by anushreeSonak**

### CODE REVIEW

File: APIAssignment/src/test/java/APIAssignments/BrandList.java [hunk 1]
```diff
@@ -0,0 +1,89 @@
...
+
+    private static final Logger logger = Logger.getLogger("BrandList.class");
+
+    @BeforeTest
+    public void getLoggerDisplay() {
+        PropertyConfigurator.configure("log4j2.properties");
+    }
+
+
+    @Test(priority = 1)
```
Comment (prashantkawa) [inline APIAssignment/src/test/java/APIAssignments/BrandList.java]: remove extra spaces

**Review by prashantkawa — CHANGES_REQUESTED** [review]

File: APIAssignment/src/test/java/APIAssignments/BrandList.java (hunk 1 above)
Comment (anushreeSonak) [reply APIAssignment/src/test/java/APIAssignments/BrandList.java]: done !

File: APIAssignment/src/test/java/APIAssignments/PutBrandList.java [hunk 2]
```diff
@@ -0,0 +1,4 @@
+package APIAssignments;
```
Comment (prashantkawa) [inline APIAssignment/src/test/java/APIAssignments/PutBrandList.java]: Remove this file if not needed

**Review by prashantkawa — CHANGES_REQUESTED** [review]

File: APIAssignment/src/test/java/APIAssignments/PutBrandList.java (hunk 2 above)
Comment (anushreeSonak) [reply APIAssignment/src/test/java/APIAssignments/PutBrandList.java]: Done !

**Review by prashantkawa — APPROVED** [review]
Approving PR

**Review by vrushalee123 — APPROVED** [review]

**Pull request merged**